.github/workflows/build.yml

```diff
@@ -345,6 +331,25 @@ jobs:
           # Render analyzer waveform tests to an offscreen buffer
           QT_QPA_PLATFORM: ${{ matrix.qt_qpa_platform }}
 
+      - name: "[Windows] Sign binaries"
+        env:
+          AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+        if: runner.os == 'Windows' && env.AZURE_TENANT_ID
+        uses: azure/azure-code-signing-action@v0.2.22
+        with:
+          azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          azure-client-secret: ${{ secrets.AZURE_CLIENT_SECRET }}
+          endpoint: https://weu.codesigning.azure.net/
+          code-signing-account-name: mixxx
+          certificate-profile-name: mixxx
+          files-folder: build
+          files-folder-filter: dll,exe
```

Comment:
mixxx.exe is signed, but none of the DLLs (with exception of vcruntime140.dll, which is provided as signed binary by Microsoft).
I'm not sure, if this is neccessary, because my Windows Firewall settings are higly customized.